webapp/root.jsx

```diff
@@ -241,6 +247,43 @@ function renderRootComponent() {
                         path=':team/logout'
                         onEnter={onLoggedOut}
                     />
+                    <Route path='settings/integrations'>
+                        <IndexRedirect to='installed'/>
+                        <Route
+                            path='installed'
+                            components={{
+                                navbar: BackstageNavbar,
+                                sidebar: BackstageSidebar,
+                                center: InstalledIntegrations
+                            }}
+                        />
+                        <Route path='add'>
+                            <IndexRoute
+                                components={{
+                                    navbar: BackstageNavbar,
+                                    sidebar: BackstageSidebar,
+                                    center: AddIntegration
+                                }}
+                            />
+                            <Route
+                                path='incoming_webhook'
+                                components={{
+                                    navbar: BackstageNavbar,
+                                    sidebar: BackstageSidebar,
+                                    center: AddIncomingWebhook
+                                }}
+                            />
+                            <Route
+                                path='outgoing_webhook'
+                                components={{
+                                    navbar: BackstageNavbar,
+                                    sidebar: BackstageSidebar,
+                                    center: AddOutgoingWebhook
+                                }}
+                            />
+                        </Route>
+                    </Route>
+>>>>>>> Added initial backstage components and InstalledIntegrations page
```

Comment:
It's even more impressive is that it actually runs fine on my local instance that way.